src/core/blockchain/near/stats.ts

```diff
@@ -30,6 +30,7 @@ export class Stats extends GenericStats<Client> {
 
         return new BigNumber(data.balance.available)
```

Comment:
this is not entirely correct. You have the following case:
- amount to keep = 100
- user has available balance 50
- user has unstaked balance 500
The user should be able to stake 500. Your version will return 450.

Basically you have to do this:
```
var aviableToDelegate = unstakedBalance
if (availableBalance > amountToKeep) aviableToDelegate += availableBalance - amountToKeep
```